app.ts

```diff
@@ -0,0 +1,32 @@
+import { Request, Response } from "express";
+const express = require('express');
+const path = require('path');
+const nunjucks = require('nunjucks');
+const session =require('express-session')
+//const { title } = require('process');
+
+const app = express();
+
+//configure nunjucks
+const appViews = path.join(__dirname, '/views');
```

Comment:
Can you add a basic HTML page to show that the project will work correctly

Reply:
> Can you add a basic HTML page to show that the project will work correctly

HTML Login page is being worked on, we just need these changes approved first. I also hotfixed the issues with app.ts if you want to review those